examples/react-native-todo-list/yarn.lock

```diff
@@ -938,6 +938,13 @@
   optionalDependencies:
```

Comment:
Is yarn necessary for React Native? We're trying to only use npm for our examples

Reply:
nope. I didn't use yarn. I delete the file. I will investigate why this file was generate.